package.json

```diff
@@ -6,23 +6,25 @@
   "dependencies": {
     "@emotion/react": "11.10.5",
     "@emotion/styled": "11.10.5",
-    "@graasp/chatbox": "1.0.0",
+    "@graasp/chatbox": "1.0.1",
     "@graasp/query-client": "0.1.2",
     "@graasp/sdk": "0.2.0",
     "@graasp/translations": "1.3.0",
     "@graasp/ui": "0.7.1",
-    "@mui/icons-material": "5.10.14",
-    "@mui/lab": "5.0.0-alpha.104",
-    "@mui/material": "5.10.14",
-    "@sentry/react": "7.29.0",
-    "@sentry/tracing": "7.29.0",
-    "@uppy/core": "2.1.6",
-    "@uppy/dashboard": "2.1.4",
-    "@uppy/drag-drop": "2.0.6",
-    "@uppy/react": "2.1.2",
-    "@uppy/status-bar": "2.1.3",
-    "@uppy/tus": "2.2.1",
-    "@uppy/xhr-upload": "2.0.7",
+    "@mui/icons-material": "5.11.0",
+    "@mui/lab": "5.0.0-alpha.117",
+    "@mui/material": "5.11.6",
+    "@sentry/react": "7.33.0",
+    "@sentry/tracing": "7.33.0",
+    "@uppy/core": "3.0.4",
+    "@uppy/dashboard": "3.2.0",
+    "@uppy/drag-drop": "3.0.1",
+    "@uppy/file-input": "3.0.1",
+    "@uppy/progress-bar": "3.0.1",
+    "@uppy/react": "3.0.2",
+    "@uppy/status-bar": "3.0.1",
+    "@uppy/tus": "3.0.5",
+    "@uppy/xhr-upload": "3.0.4",
```

Comment:
Does this major update break anything in our code?

Reply:
Only change I had to make was related to [@uppy/xhr-upload `metaFields` which was renamed to `allowedMetaFields`](https://uppy.io/docs/migration-guides.html#uppy-tus-uppy-aws-s3-uppy-xhr-upload)